src/channel_internal.c

```diff
@@ -139,17 +151,29 @@ static void execute_callbacks(void* context)
         CHANNEL_OP* channel_op = (CHANNEL_OP*)context;
         CHANNEL_CALLBACK_RESULT result = channel_op->result; // local copy to make sure both callbacks are called with the same result
 
+        LOGGER_LOG(LOG_LEVEL_VERBOSE, T_PTR_VALUE_OR_NULL(channel_op->channel_internal->log_context),
+            "Executing callbacks for"
+            " pull_correlation_id = %" PRI_RC_STRING ""
+            ", push_correlation_id = %" PRI_RC_STRING ""
+            ", callback_result=%" PRI_MU_ENUM "",
```

Comment:
>= [](http://example.com/codeflow?start=30&length=1)

either have spaces EVERYWHERE around "=" or nowhere.

I would suggest following the repo convention if such a thing exists / would keep the logging consistent across all users of c-util. I suspect geo-ification.

Reply:
Good catch, adding spaces

Reply:
You misspelled "remove"

Reply:
Seems like c_util has presence of "= %" and "=%".

I like having the spaces so I will add the spaces.